### PR TITLE
Reorder dataservice publish form

### PR DESCRIPTION
**Pull request by geoffreyaldebert**
- [x] update: translation for dataservice
- [x] reorder fields
- [x] finish checkup for dataservice

### CODE REVIEW

**Comment by nicolaskempf57** [comment]
I think that the reorder won't work because of the way the datasets are fetched.
@ThibaudDauce knows more about this

**Comment by geoffreyaldebert** [comment]
that is not the reorder of the datasets sorry but the reorder of fields in the form. this one os testednand work for me. reordering datasets is another subject not in this PR

**Comment by ThibaudDauce** [comment]
@geoffreyaldebert  Just a quick look and I don't understand why there is a new `AdminDataservicesTable` file? :-) But I will pull it and test it locally to try to understand where it is used…

File: components/AdminDataservicesTable/AdminDataservicesTable.vue
Comment (ThibaudDauce) [inline components/AdminDataservicesTable/AdminDataservicesTable.vue:1]: I'm not sure to understand where this is used…

**Review by ThibaudDauce — APPROVED** [review]

**Pull request merged**